recipes/_compute_base_config.rb

```diff
@@ -55,6 +55,12 @@
   end
 end
 
+# Only run FSx on centos for now
+if node['platform'] == 'centos'
+  # Install FSx
```

Comment:
remove this line, installation needs to be moved in base_install.rb

Reply:
done